docs/examples/tag/dontInteractive.js

```diff
@@ -0,0 +1,11 @@
+// @flow strict
+import { type Node } from 'react';
+import { Flex, Tag } from 'gestalt';
+
+export default function Example(): Node {
+  return (
+    <Flex alignItems="center" justifyContent="center" height="100%" width="100%">
+      <Tag disabled onRemove={() => {}} text="Visit Pinterest" />
```

Comment:
This Tag is no interactive, we should add a TapArea around it?

Reply:
sure, that's a good idea